src/backend/src/utils/reimbursement-requests.utils.ts

```diff
@@ -15,12 +15,56 @@ export interface ReimbursementProductCreateArgs {
   wbsElementId: number;
 }
 
+export interface ReimbursementReceiptCreateArgs {
+  name: string;
+  googleFileId: string;
+}
+
 export interface ReimbursementProductCreateArgs {
   name: string;
   cost: number;
   wbsElementId: number;
 }
 
+/**
+ * This function removes any deleted receipts and adds any new receipts
+ * @param receipts the new list of receipts to compare against the old ones
+ * @param currentReceipts the current list of receipts on the request that's being edited
+ * @param reimbursementRequestId the id of the reimbursement request that's being edited
+ */
+export const updateReceiptPictures = async (
+  receipts: ReimbursementReceiptCreateArgs[],
+  currentReceipts: Receipt[],
+  reimbursementRequestId: string
+) => {
+  if (receipts.length > 0) {
```

Comment:
make a guard statement i.e if receipts.length === 0 return

Reply:
this is just so it only tries to create new receipts if any new receipts were submitted (same with only trying to delete receipts if there are existing receipts)

Reply:
@Peyton-McKee

Reply:
yeah but the entire thing is now indented if you make a guard statement you can just check for that and return instead of having everything in a big if statement